content/en/docs/distributions/gke/deploy/upgrade.md

```diff
@@ -1,31 +1,103 @@
 +++
 title = "Upgrade Kubeflow"
 description = "Upgrading your Kubeflow installation on Google Cloud"
-weight = 5
+weight = 6
 +++
 
 ## Before you start
 
 To better understand upgrade process, you should read the following sections first:
 
-* [Understanding the deployment process for management cluster](../management-setup#understanding-the-deployment-process)
-* [Understanding the deployment process for Kubeflow cluster](../deploy-cli#understanding-the-deployment-process)
+* [Understanding the deployment process for management cluster](/docs/gke/deploy/management-setup#understanding-the-deployment-process)
```

Comment:
nit: gke docs are now under `/docs/distributions/gke/...`

please fix all other usages

Reply:
Applied this changes across `deploy/`